docker/generic/start_proxy.py

```diff
@@ -135,6 +135,28 @@ def make_argparser():
         manually. Valid schemes are `http`, `https`, `grpc`, and `grpcs`.
         '''.format(backend=DEFAULT_BACKEND))
 
+    parser.add_argument(
+        '--enable_backend_address_override',
+        action='store_true',
+        help='''
+        Backend addresses can be specified using either the --backend flag
+        or the `backend.rule.address` field in the service configuration.
+        
+        By default, the `backend.rule.address` will take priority over 
+        the --backend flag for each individual operation.
+        
+        Enable this flag if you want the --backend flag to take priority
+        instead. This is useful if you are developing on a local workstation.
+        Then you use a the same production service config but override the
+        backend address via the --backend flag for local testing.
+        
+        Note: Only the address will be overridden.
+        All other components of `backend.rule` will still apply 
+        (deadlines, backend auth, etc).
+        Consider setting --non_gcp as well to override the 
+        backend auth configuration.
```

Comment:
--non_gcp is reverent here. I suggest to remove these two lines

Reply:
Done